doc/newsboat.asciidoc

```diff
@@ -1306,6 +1306,32 @@ open dialogs by selecting them and pressing "Ctrl-X".
 
 include::chapter-podcasts.asciidoc[]
 
+=== Media Content
+
+Since articles can link to various media content such as _audio_, _video_ and _images_, you may want to open those files with a dedicated application instead of a web browser.
+
+As an alternative to Podboat, media players such as `mpv` can stream content directly when given a URL as an argument. For podcasts, Newsboat always assign the main audio file to link [1]. Using the technique described in <<_macro_support,Macro Support>>, a specific link can be opened with the command of your choice, right from the article view (no need to reach for the URL view for this task). Instead of calling `open-in-browser`, the operations `one` to `zero` (numbers written textually) can be used to specify _which_ link to open.
+
+So, to stream a podcast from the article view, you can use a macro such as:
+
+```
+macro 1 set browser mpv; one; set browser firefox
+```
+
+While listening, you might also want to keep using newsboat at the same time. There are many ways to achieve that. One of them is described in the link:faq.html#_after_i_open_a_link_in_the_browser_newsboat_stays_unresponsive_until_the_browser_is_closed[FAQ], another is to add `setsid -f` at the beginning of the command to fork the external application to a new process, independant of Newsboat. In addition to that, you probably want to open the media player in a dedicated terminal. The following macro does just that:
```

Comment:
> There are many ways to achieve that. One of them is described in the link:faq.html#_after_i_open_a_link_in_the_browser_newsboat_stays_unresponsive_until_the_browser_is_closed[FAQ], another is to add `setsid -f` at the beginning of the command to fork the external application to a new process, independant of Newsboat.

It's better to consolidate the information in one place, rather than keep one piece in FAQ and another here. I'd just add `setsid` to the FAQ item, but I'm open to other ideas.

Also please note that `setsid` (the command) seem to be Linux-specific. It's missing at least on FreeBSD and macOS (although available through Homebrew via `util-linux` package). Newsboat is portable across Unix-likes, so please mark any non-portable advice as such.

Reply:
I use Newsboat on Linux only so I forgot about portability. I like `setsid -f` because it makes for a clean, readable macro command. I wanted the examples to be as simple as possible. Another way, `firefox </dev/null &>/dev/null &`, is explained [here](https://superuser.com/a/178592), though I cannot tell if it's portable or not.

I'm not sure either about the FAQ, maybe I can add both methods as alternatives to "wrap in a script". Maybe i'm being lazy, but I didn't want to create a script when everything can be self-contained in the macro.

If we proceed with `setsid -f` in the docs, I will make sure to rewrite this section and point out it's non-portable.

Reply:
> Maybe i'm being lazy, but I didn't want to create a script when everything can be self-contained in the macro.

Nah, you're fine; it's the FAQ that is overly complicated. I'll file an issue to simplify that after this PR gets merged.

> Another way, firefox </dev/null &>/dev/null &, is explained here, though I cannot tell if it's portable or not.

It's not portable. Newsboat executes commands via `/bin/sh`, which only guarantees support for POSIX shell constructs, and [`&>` is not one of them](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07). But you can emulate `&>/dev/null` with `>/dev/null 2>/dev/null`, of course.

> I like setsid -f because it makes for a clean, readable macro command. I wanted the examples to be as simple as possible. … If we proceed with setsid -f in the docs, I will make sure to rewrite this section and point out it's non-portable.

`CMD &` is even cleaner, I think. The double-fork is already achieved by Newsboat calling /bin/sh which calls `CMD`, and `setsid()` (the syscall) is irrelevant since MPV and Alacritty are GUI apps that don't care about their controlling TTYs. This makes the two approaches equal, so if there are no other reasons to use `setsid`, I'd go with `&`.

Reply:
I ended up with `>/dev/null 2>&1` and `&`. Portable, simple, works great.
Thanks for the hints, was really helpful!